docs/disk-caching/DESIGN.md

```diff
@@ -1,53 +1,63 @@
 # Disk Caching Design [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-This document explains some basic assumptions and design approach, limits of the disk caching feature. If you're looking to get started with disk cache, we suggest you go through the [getting started document](https://github.com/minio/minio/blob/master/docs/disk-caching/README.md) first.
+This topic explains the attributes and requirements, design approach, and limits of the disk caching feature. To get started with the disk caching feature, see the [Disk Cache Quickstart Guide](https://github.com/minio/minio/blob/master/docs/disk-caching/README.md).
 
-## Command-line
+- [General Command-line Usage](#general-command-line-usage) 
+- [Attributes and Requirements](#attributes-and-requirements) 
+- [Behavior](#behavior) 
+- [Limits](#limits) 
+
+## <a name="general-command-line-usage"></a>1. General Command-line Usage
+
+The following shows the general usage for Minio Server:
 
 ```
 minio server -h
 ...
 ...
   CACHE:
-     MINIO_CACHE_DRIVES: List of mounted cache drives or directories delimited by ";"
-     MINIO_CACHE_EXCLUDE: List of cache exclusion patterns delimited by ";"
-     MINIO_CACHE_EXPIRY: Cache expiry duration in days
-     MINIO_CACHE_MAXUSE: Maximum permitted usage of the cache in percentage (0-100).
+     MINIO_CACHE_DRIVES: List of mounted cache drives or directories delimited by ";".
+     MINIO_CACHE_EXCLUDE: List of cache exclusion patterns delimited by ";".
+     MINIO_CACHE_EXPIRY: Cache expiry duration in days.
+     MINIO_CACHE_MAXUSE: Maximum permitted usage of the cache as a percentage (0-100).
 ...
 ...
 
-  7. Start minio server with edge caching enabled on '/mnt/drive1', '/mnt/drive2' and '/mnt/export1 ... /mnt/export24',
-     exclude all objects under 'mybucket', exclude all objects with '.pdf' as extension
-     with expiry up to 40 days.
+  7. Start Minio Server with edge caching enabled on '/mnt/drive1', '/mnt/drive2' and '/mnt/export1 ... /mnt/export24',
```

Comment:
We need to copy over the mc `get / set` config per section 2 of the disk cache quickstart guide